lib/QubitCsvTransform.class.php

```diff
@@ -134,13 +144,13 @@ public function addRowToMySQL($sortorder)
         }
     }
 
-    public function numberedFilePathVariation($filename, $number)
+    public static function numberedFilePathVariation($filename, $number)
     {
         $parts = pathinfo($filename);
         $base = $parts['filename'];
         $path = $parts['dirname'];
 
-        return $path.'/'.$base.'_'.$number.'.'.$parts['extension'];
+        return $path.'/'.$base.'_'.sprintf('%04d', $number).'.'.$parts['extension'];
```

Comment:
Suggestion: Could use `sprintf()` for the whole string, and use `$parts['filename']` and `$parts['dirname']` directly instead of assigning one off variables.

Reply:
Good call... that definitely ends up looking cleaner.